package/MDAnalysis/topology/ParmEdParser.py

```diff
@@ -0,0 +1,317 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4 
+#
+# MDAnalysis --- https://www.mdanalysis.org
+# Copyright (c) 2006-2017 The MDAnalysis Development Team and contributors
+# (see the file AUTHORS for the full list of names)
+#
+# Released under the GNU Public Licence, v2 or any higher version
+#
+# Please cite your use of MDAnalysis in published work:
+#
+# R. J. Gowers, M. Linke, J. Barnoud, T. J. E. Reddy, M. N. Melo, S. L. Seyler,
+# D. L. Dotson, J. Domanski, S. Buchoux, I. M. Kenney, and O. Beckstein.
+# MDAnalysis: A Python package for the rapid analysis of molecular dynamics
+# simulations. In S. Benthall and S. Rostrup editors, Proceedings of the 15th
+# Python in Science Conference, pages 102-109, Austin, TX, 2016. SciPy.
+# doi: 10.25080/majora-629e541a-00e
+#
+# N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and O. Beckstein.
+# MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
+# J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
+#
+
+"""
+ParmEd topology parser
+===================
+
+Converts a ParmEd_ structure into a Topology.
+
+.. _ParmEd: https://parmed.github.io/ParmEd/html/index.html
+
+Example
+-------
+
+>>> import parmed as pmd
+>>> import MDAnalysis as mda
+>>> from MDAnalysis.tests.datafiles import GRO
+>>> pgro = pmd.load_file(GRO)
+>>> mgro = mda.Universe(pgro)
+>>> parmed_subset = mgro.select_atoms('resname SOL').write(file_format='PARMED')
+
+Classes
+-------
+
+.. autoclass:: ParmEdParser
+   :members:
+   :inherited-members:
+
+"""
+from __future__ import absolute_import, division
+from six.moves import range
+from six import raise_from
+
+import logging
+import functools
+from math import ceil
+import numpy as np
+
+from ..lib.util import openany
+from .guessers import guess_atom_element
+from .base import TopologyReaderBase, change_squash
+from ..core.topologyattrs import (
+    Atomids,
+    Atomnames,
+    AltLocs,
+    ChainIDs,
+    Atomtypes,
+    Occupancies,
+    Tempfactors,
+    Elements,
+    Masses,
+    Charges,
+    Resids,
+    Resnums,
+    Resnames,
+    Segids,
+    GBScreens,
+    SolventRadii,
+    NonbondedIndices,
+    RMins,
+    Epsilons,
+    RMin14s,
+    Epsilon14s,
+    Bonds,
+    UreyBradleys,
+    Angles,
+    Dihedrals,
+    Impropers,
+    CMaps
+)
+from ..core.topology import Topology
+
+logger = logging.getLogger("MDAnalysis.topology.ParmEdParser")
+
+# >>> import mendeleev as mv
+# >>> {i:mv.element(i).symbol for i in range(1, 100)}
+Z2SYMB = {1: 'H',                                                    2: 'He', 
```

Comment:
Can't you generate this table from `coordinates.ParmEd.SYMB2Z` ? 

Furthermore, perhaps just putting the periodic table into `lib.tables` might be a better place anyway?